packages/gestalt/src/Table/TableSortableHeaderCell.js

```diff
@@ -1,9 +1,11 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
+/* eslint-disable import/no-relative-parent-imports */
```

Comment:
@simkessy We have this rule to avoid `../` in our codebase. So I would recommend to keep the flat structure as we currently have it and not nest components inside of `Table`

Reply:
Why do we want to keep a flat structure? Especially as we're adding more complicated components like Tables, DatePickers, Typeaheads. To have all those pieces flat seems like it would make organization challenging.  

Comparing with other UI libraries:
https://github.com/mui-org/material-ui/tree/next/packages/material-ui/src
https://github.com/ant-design/ant-design/tree/master/components